src/Persistence/AbstractQuery.php

```diff
@@ -53,11 +57,19 @@ public function __construct(Model $model)
         $this->limit = $model->limit;
     }
 
+    /**
+     * Find and return data from record with $id or NULL if none found.
+     */
     public function find($id): ?array
     {
         return $this->whereId($id)->getRow();
     }
 
+    /**
+     * Setup query as selecting list of $fields or all if $field = NULL.
```

Comment:
the only valueable/not implied comment part is `Setup query` only - this is bad design